de.metas.business/src/main/java/de/metas/phonecall/service/PhonecallScheduleRepository.java

```diff
@@ -69,7 +72,8 @@ public void save(@NonNull final PhonecallSchedule schedule)
 		scheduleRecord.setIsCalled(schedule.isCalled());
 		scheduleRecord.setIsOrdered(schedule.isOrdered());
 
-		scheduleRecord.setSalesRep_ID(UserId.toRepoId(schedule.getSalesRepId()));
+		final I_C_BPartner bpartner = Services.get(IBPartnerDAO.class).getById(schedule.getBpartnerAndLocationId().getBpartnerId());
```

Comment:
IMHO this is hidden functionality.
The SalesRepId shall come from PhonecallSchedule object.

Reply:
Done. Created a getter in BPartnerBL and used that.